pkg/apis/operator/v1alpha1/openshiftpipelinesascode_validation.go

```diff
@@ -79,6 +79,9 @@ func (aps AdditionalPACControllerConfig) validate(logger *zap.SugaredLogger, pat
 	}
 
 	defaultPacSettings := pacSettings.DefaultSettings()
+	if aps.Settings == nil {
+		aps.Settings = map[string]string{}
```

Comment:
why don't you use a constructor app.NewSettings() blah

Reply:
I need to check the operator code how to initialize these things

Right now i have move this to defaulting code instead of validation

Reply:
Yes agree :+1: